### PR TITLE
Move the new app to a separate folder and entrypoint

### PR DESCRIPTION
**Pull request by patrys**
This cleans up the structure and gives the new app its own namespace both in JS code and in URLs.

### Pull Request Checklist

(Please keep this section. It will make maintainer's life easier.)

1. [x] Privileged views and APIs are guarded by proper permission checks.
1. [x] All visible strings are translated with proper context.
1. [x] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] Database queries are optimized and the number of queries is constant.
1. [ ] The changes are tested.
1. [x] The code is documented (docstrings, project documentation).
1. [x] Python code quality checks pass: `pycodestyle`, `pydocstyle`, `pylint`.
1. [ ] JavaScript code quality checks pass: `eslint`.

### CODE REVIEW

File: templates/dashboard/next.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+<html lang="{{ LANGUAGE_CODE }}" class="no-js">
+  <head>
+    <title>
+      {% block title %}
+        {% trans "Dashboard - Saleor" context "Dashboard default page title" %}
+      {% endblock %}
+    </title>
+    {% block meta %}
+      <meta charset="utf-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">
```
Comment (dominik-zeglen) [inline templates/dashboard/next.html]: I'm not sure if blocking site scaling is good idea

File: saleor/static/dashboard-next/js/misc.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+export const screenSizes = {
+  sm: 600,
+  md: 992,
+  lg: 1200
+};
+
+export function createQueryString(currentQs, newData) {
```
Comment (dominik-zeglen) [inline saleor/static/dashboard-next/js/misc.js]: Is mixing constants and utility functions in one file a good idea?

File: templates/dashboard/next.html (hunk 1 above)
Comment (patrys) [reply templates/dashboard/next.html]: That's our current template and we do want to disable scaling for responsive design and modal positioning to work properly.

File: saleor/static/dashboard-next/js/misc.js (hunk 2 above)
Comment (patrys) [reply saleor/static/dashboard-next/js/misc.js]: I'll get rid of both eventually, creating two temporary files did not seem like a better idea.

**Pull request merged**